project.clj

```diff
@@ -11,4 +11,5 @@
   :global-vars {*warn-on-reflection* true}
   :dependencies [[org.clojure/tools.reader "1.0.0-beta3"]]
   :profiles {:dev {:dependencies [[org.clojure/clojure "1.9.0-alpha13"]
-                                  [org.clojure/test.check "0.9.0"]]}})
+                                  [org.clojure/test.check "0.9.0"]]}}
+  :monkeypatch-clojure-test false)
```

Comment:
So, I've found what this does, but I'm a little unclear as to why it's necessary
```
  ;; In order to support the `retest` task, Leiningen must monkeypatch the
  ;; clojure.test library. This disables that feature and breaks `lein retest`.
  :monkeypatch-clojure-test false
```
- from [leiningen sample project.clj]
(https://github.com/technomancy/leiningen/blob/master/sample.project.clj#L346-L348)
```